docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -307,8 +307,16 @@ You can make further modifications to a saved filter view later using kebab menu
 
 ## Create an SLO from Metrics page
 
-:::note Coming soon
-You'll be able to create SLOs from Metrics Explorer.
+To create an SLO from the **Metrics** page:
+
+1. Click xyz...
+
+You can use [metrics operators](/docs/metrics/metrics-operators) for metrics-based SLOs. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO.
+
+As an example, a pure selector query with no operators is `_sourceHost=stag-monitor-manager metric=CPU_Total`. That same query using the `sum` operator would be `_sourceHost=stag-monitor-manager metric=CPU_Total | sum`.
+
```

Comment:
```That same query using the `sum` operator would be `_sourceHost=stag-monitor-manager metric=CPU_Total | sum`.```

To use operator we have a constraint that the user should provide a quantization after the selector. Example: `_sourceHost=stag-monitor-manager metric=CPU_Total | quantize to 1m using avg | sum`

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/2725#discussion_r1289237320